janus_core/calculations/eos.py

```diff
@@ -104,8 +100,7 @@ class EoS(BaseCalculation):
 
     def __init__(
         self,
-        struct: Atoms | None = None,
-        struct_path: PathLike | None = None,
+        struct: Atoms | PathLike,
```

Comment:
```suggestion
        struct: MaybeSequence[Atoms] | PathLike,
```

Reply:
Equation of state doesn't (currently) accept multiple input structures, so I think just `Atoms` is correct here